docs/pages/setup/helm-deployments/aws.mdx

```diff
@@ -530,6 +532,53 @@ Here's an example where we set the chart to use 3 replicas:
   To change `chartMode`, `clusterName`, or any `aws` settings, you must first uninstall the existing chart and then install a new version with the appropriate values.
 </Admonition>
 
+### Autoscaling
+
+In order to reduce DynamoDB costs you might want to enable DynamoDB autoscaling.
+This can be done manually or delegated to Teleport.
+
+This is step is usually done after a successful Teleport deployment, once you have gathered some data
+about Teleport's DynamoDB usage and know what regular usage looks like and how autoscaling should be tuned.
+See [the DynamoDB autoscaling section](../reference/backends.mdx#dynamodb-autoscaling) for more details.
+
+To go further you have to know the desired read/write minimum, maximum and target capacity.
```

Comment:
What does "To go further" mean here? Can we combine this sentence with the previous paragraph, which already says that autoscaling is usually set up after gathering some data?